client/question/services/FeedbackGeneratorService.js

```diff
@@ -157,11 +173,11 @@ tie.factory('FeedbackGeneratorService', [
      * @param {string} language
      * @returns {string | null} Text to be appended to feedback.
      */
-    var _getHumanReadableRuntimeFeedback = function(errorString, language) {
+    var _getFriendlyRuntimeFeedback = function(errorString, language) {
       var result = null;
-      RUNTIME_ERROR_FEEDBACK_MESSAGES[language].forEach(function(check) {
-        if (check.checker(errorString)) {
-          result = check.generateMessage(errorString);
+      FRIENDLY_RUNTIME_ERROR_TRANSLATIONS[language].forEach(function(check) {
```

Comment:
I think I saw some forEach loops above where the dummy variable was 'error' rather than 'check', e.g. https://github.com/google/tie/pull/698/files#diff-b5e458ce6bd4b4d8d91bb228b3182735R111 .  We might want to standardize.

(I think this was an issue in the old version of the code, too.)

Reply:
@rabidbit FYI, this comment hasn't been replied to.

Reply:
Will change `WRONG_LANGUAGE_ERRORS.python.forEach(function(error)` to `WRONG_LANGUAGE_ERRORS.python.forEach(function(check)`